Tests/ArgumentParserUnitTests/HelpGenerationTests.swift

```diff
@@ -505,6 +505,49 @@ extension HelpGenerationTests {
         """
     )
   }
+
+  struct AllValues: ParsableCommand {
+    enum Manual: Decodable, ExpressibleByArgument {
```

Comment:
Looks like this just needs to be a RawRepresentable enum.
```suggestion
    enum Manual: String, ExpressibleByArgument {
```

Reply:
Ah, this is intentionally not a String to test that manually implemented `allValueStrings` works

Reply:
Int works too, then. It’s not getting Decodable conformance synthesis without the raw form. (I suppose you could also implement `init(from:)`)

Reply:
Ah I overlooked this since 5.5 synthesized the conformance locally (somehow)